SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -38,18 +38,21 @@ async def get(self):
             makenv_process = self._launch_makenv()
             SwanCustomEnvironmentsApiHandler.makenv_process = makenv_process
             log_file = open(self.LOG_FILE, "w")
+        elif makenv_process.poll() == 1:
+            # If the process has finished, open the log file in read mode to check for errors
```

Comment:
An error return code could also be different than 1 but still indicate some error. It has to be different than zero and different than None.